paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/utils/FlowControllerTestRunner.kt

```diff
@@ -94,9 +103,75 @@ private fun runFlowControllerTest(
 
     scenario.moveToState(Lifecycle.State.RESUMED)
 
-    val testContext = FlowControllerTestRunnerContext(scenario, flowController)
+    val testContext = FlowControllerTestRunnerContext(manualIdlingResource, scenario, flowController)
     block(testContext)
 
     assertThat(countDownLatch.await(5, TimeUnit.SECONDS)).isTrue()
     scenario.close()
 }
+
+private class TestFlowControllerWrapper(
+    private val manualIdlingResource: ManualIdlingResource,
+    private val flowController: PaymentSheet.FlowController
+) : PaymentSheet.FlowController {
```

Comment:
```suggestion
) : PaymentSheet.FlowController by flowController {
```

Will allow you to only override the things you want to.